### PR TITLE
chore: Rename various stuff to "main"

### PR DESCRIPTION
**Pull request by Jiralite**
**Please describe the changes this PR makes and why it should be merged:**
With the default branch now named as "main", I think it would be suitable to correct some links!

In the links section, the documentation was pointing to master branch (instead of main) but a later link was pointing to stable. I just made the links here match.

Let me know if I should amend anything!

**Status and versioning classification:**
- Code changes have been tested against the Discord API, or there are no code changes
- I know how to update typings and have done so, or typings don't need updating
- This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ assignees: ''
 
 <!--
 Remove the comment and fill out the commit hash if this applies to you:
-(While it's not a requirement to test your issue on the master branch, it would make fixing the problem a lot easier for us, so please do so if possible.)
+(While it's not a requirement to test your issue on the main branch, it would make fixing the problem a lot easier for us, so please do so if possible.)
 
-- I have also tested the issue on latest master, commit hash: `xxx`
+- I have also tested the issue on latest main commit hash: `xxx`
```
Comment (iCrawl) [inline .github/ISSUE_TEMPLATE/bug_report.md:37]:
```suggestion
- I have also tested the issue on latest main; Commit hash: `xxx`
```

**Comment by NotSugden** [comment]
https://github.com/discordjs/discord.js/blob/cfde127072ae06b63f17ba19b3e0d28fddef97d6/src/client/Client.js#L616

**Comment by Jiralite** [comment]
>   * @see {@link https://discord.js.org/#/docs/collection/master/class/Collection}

Oh gosh, thank you... might help if I actually keep my branch updated haha!

**Comment by NotSugden** [comment]
perhaps https://github.com/discordjs/discord.js/blob/cfde127072ae06b63f17ba19b3e0d28fddef97d6/src/sharding/ShardClientUtil.js#L9 and other instances like this aught to be changed (possibly in a seperate PR)

**Comment by Jiralite** [comment]
I left the sharding stuff alone because I too thought that would probably be best done in a separate pull request... This was just for the rename of the default branch, so. Yeah.

**Review by iCrawl — APPROVED** [review]

**Pull request merged**